packages/babel-generator/src/index.js

```diff
@@ -58,7 +58,6 @@ function normalizeOptions(code, opts, tokens): Format {
     compact: opts.compact,
     minified: opts.minified,
     concise: opts.concise,
-    quotes: opts.quotes || findCommonStringDelimiter(code, tokens),
```

Comment:
Do we want to keep `findCommonStringDelimiter`? If we do we then this pr can just be removing `opts.quotes` rather than defaulting to double.